CMakeLists.txt

```diff
@@ -31,19 +31,16 @@ if(POLICY CMP0074)
   cmake_policy(SET CMP0074 NEW)
 endif()
 
+if(POLICY CMP0135)
```

Comment:
Maybe a comment on what this magic number, 135, means?

Reply:
done